wolfcrypt/src/logging.c

```diff
@@ -1248,7 +1248,7 @@ unsigned long wc_GetErrorNodeErr(void)
 
     if (ERRQ_LOCK() != 0) {
         WOLFSSL_MSG("Lock debug mutex failed");
-        return BAD_MUTEX_E;
+        return -BAD_MUTEX_E;
```

Comment:
I think this is already fixed here: https://github.com/wolfSSL/wolfssl/pull/6195